app/renderer/components/styles/global.js

```diff
@@ -214,6 +214,7 @@ const globalStyles = {
     zindexWindowIsPreview: '1100',
     zindexDownloadsBar: '1000',
     zindexTabs: '1000',
+    zindexTabsAudioTopBorder: '10001',
```

Comment:
Is it not `1001` instead of `10001`?

Reply:
Confirmed this was addressed on https://github.com/brave/browser-laptop/pull/10691/files#diff-f14732b3ea914a26c2215676e7208907R217

On the commit: https://github.com/brave/browser-laptop/commit/f4780d31478ed4f1091a86f1f19f4c983dd82590#diff-f14732b3ea914a26c2215676e7208907R217